components/calendar/src/julian.rs

```diff
@@ -65,37 +84,35 @@ impl Calendar for Julian {
         _largest_unit: DateDurationUnit,
         _smallest_unit: DateDurationUnit,
     ) -> DateDuration<Self> {
-        DateDuration::new(
-            date1.0.year.0 - date2.0.year.0,
-            date1.0.year.0 - date2.0.year.0,
-            0,
-            u8::from(date1.0.day) as i32 - u8::from(date2.0.day) as i32,
-        )
+        date1.0.until(date2.0, _largest_unit, _smallest_unit)
     }
 
     /// The calendar-specific year represented by `date`
     fn year(&self, date: &Self::DateInner) -> types::Year {
-        date.0.year.into()
+        crate::gregorian::year_as_gregorian(date.0.year)
```

Comment:
nit: drop in a comment saying that the Julian year has the same era scheme as Gregorian

Reply:
Done